js/pluralizer.js

```diff
@@ -0,0 +1,4 @@
+let thing = 'dog';
+let count = 5;
+
+console.log(count + " "+ thing)
```

Comment:
in english the plural of 'dog' is 'dogs' , we should conditionally be adding the 's'